src/web/_market/portal/meta.json

```diff
@@ -11,39 +11,37 @@
   "versionDisplay": "portal",
   "installMatcher": "best-match",
   "mavenArtifacts": [
-    {
-      "key": "portal-10",
-      "name": "Portal App (10.0)",
-      "groupId": "ch.ivyteam.ivy.project.portal",
-      "artifactId": "portal-app",
-      "type": "zip"
-    },
-    {
-      "name": "Portal Guide (10.0)",
-      "groupId": "ch.ivyteam.ivy.project.portal",
-      "artifactId": "portal-guide",
-      "type" : "zip",
-      "doc": true
-    },
     {
       "key": "portal",
       "name": "Portal App",
       "groupId": "com.axonivy.portal",
+      "archivedArtifact": [
+        {
+          "groupId": "ch.ivyteam.ivy.project.portal",
```

Comment:
you don't need the artifactId?

Reply:
Thanks for your  mention, I would update that property also

Reply:
can you design it that way, that archivedArtifacts are fully described. GroupId and ArtifactId. If someone want to change the artifactId too, he can do it. We have this use case in the doc-factory for older versions I guess

Reply:
Absolutely, I checked x-connector and found that we changed the group ID and artifact ID from Twitter to X. I will update it soon.